test/cpp/end2end/xds/xds_override_host_end2end_test.cc

```diff
@@ -449,34 +482,57 @@ TEST_P(OverrideHostTest, ClusterGoneHostStays) {
       GetAffinityCookieHeaderForBackend(DEBUG_LOCATION, 1, kNumEchoRpcs / 3));
 }
 
-TEST_P(OverrideHostTest, EnablePerRoute) {
-  CreateAndStartBackends(2);
+TEST_P(OverrideHostTest, EnabledInRouteConfig) {
+  CreateAndStartBackends(1);
```

Comment:
I think this test needs two endpoints.  With only one endpoint, we're not actually checking that SSA is working, because the traffic would always go to the same endpoint even without the feature enabled.